examples/chef/common/DataModelUtils.cpp

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
please avoid "filler" names such as utils/helpers/support/core and similar, the concern being that they end up being a "catch-all for everything" and encourage coupling.

In this case, these files seem to handle endpoint device types, so name the files something that reflects that. Maybe `DeviceTypes.h` or `DeviceTypeHandling` or similar. This applies to the namespace as well: `DataModelUtils` can mean anything.